### PR TITLE
Openshift cnv branding

### PR DESCRIPTION
**Pull request by rawagner**
https://bugzilla.redhat.com/show_bug.cgi?id=1653769

### CODE REVIEW

**Comment by rawagner** [comment]
@mareklibra I have a few questions
 - new logo is created for Openshift Virtualization - I combined the existing openshift logo and part of OKD Virt - SVG header and Text and adjusted X/Y values. Is this a correct approach ?

- added new openshiftvirtualization branding - question is how to adjust the downstream build to set the server flag to `openshiftvirtualization` value ?

**Comment by rawagner** [comment]
branding

File: frontend/public/branding.ts [hunk 1]
```diff
@@ -8,6 +8,8 @@ const getProductName = () => {
       return 'OpenShift Online';
     case 'okdvirt':
       return 'OKD Virtualization';
+    case 'openshiftvirt':
+      return 'OpenShift Virtualization';
```
Comment (mareklibra) [inline frontend/public/branding.ts]: ... and change the text here

File: frontend/public/components/masthead.tsx [hunk 2]
```diff
@@ -59,12 +60,20 @@ export const getBrandingDetails = () => {
       productTitle = <React.Fragment>Red Hat<sup>&reg;</sup> OpenShift Dedicated</React.Fragment>;
       break;
     case 'okdvirt':
-      backgroundImg = true;
+      backgroundImg = false;
```
Comment (mareklibra) [inline frontend/public/components/masthead.tsx]: This is "our" `case`

File: cmd/bridge/main.go [hunk 3]
```diff
@@ -148,12 +148,13 @@ func main() {
 	switch branding {
 	case "okd":
 	case "okdvirt":
+	case "openshiftvirt":
```
Comment (mareklibra) [inline cmd/bridge/main.go]: I've added the `okdvirt` on previous line for "out" customization.

I think you just need to `update` the already existing `.svg` file.

**Comment by mareklibra** [comment]
> new logo is created for Openshift Virtualization - I manually combined the existing openshift logo and part of OKD Virt logo - SVG header and Text and adjusted X/Y values in header. Is this a correct approach ?

No need for new file. Just update the existing one and change the text value.

>    added new openshiftvirtualization branding - question is how to adjust the downstream build to set the server flag to openshiftvirtualization value ?

It is requested by the `--branding` parameter of the `bridge` process. Already done.
I don't see difference between upstream/downstream logo.

Anyway, if this is required, we can either
- patch downstream
- introduce new branding and just run the container's process with different parameter

The later option is much simpler.

**Comment by rawagner** [comment]
> > new logo is created for Openshift Virtualization - I manually combined the existing openshift logo and part of OKD Virt logo - SVG header and Text and adjusted X/Y values in header. Is this a correct approach ?
> 
> No need for new file. Just update the existing one and change the text value.

Actually the text (Virtualization) is the same, logo/image changed.
> 
> > added new openshiftvirtualization branding - question is how to adjust the downstream build to set the server flag to openshiftvirtualization value ?
> 
> It is requested by the `--branding` parameter of the `bridge` process. Already done.
> I don't see difference between upstream/downstream logo.

upstream logo 
upstream

downstream logo
branding

> 
> Anyway, if this is required, we can either
> 
> * patch downstream
> * introduce new branding and just run the container's process with different parameter
> 
> The later option is much simpler.
I agree that later is simpler and i would go with this approach. New branding is added in this PR (openshiftvirt for downstream, okdvirt for upstream)

**Comment by rawagner** [comment]
@mareklibra sorry for not being clear in the beginning. Added bugzilla link to PR description. We need Openshift branding for downstream build beside already existing OKD branding for upstream

**Comment by mareklibra** [comment]
> @mareklibra sorry for not being clear in the beginning. Added bugzilla link to PR description. We need Openshift branding for downstream build beside already existing OKD branding for upstream

Thank you, I missed that.
Then adding new branding here is the easiest solution.
I can handle downstream.

**Comment by mareklibra** [comment]
Branding can be set via ConfigMap.
The [operator](https://github.com/kubevirt/web-ui-operator/pull/22) reflects that.

**Pull request merged**